docs/design/datamover.md

```diff
@@ -198,7 +184,7 @@ spec:
     copyMethod: None
 ```
 
-Similarly, when a DataMoverRestore CR gets created, controller will create a `ReplicationDestination` CR in the protected namespace. VolSync controller copies the PVC data from the restic repository to the protect namespace, which then gets transferred to the user namespace by the controller.
+Similarly, when a VolumeSnapshotRestore CR gets created, controller will create a `ReplicationDestination` CR in the protected namespace. VolSync controller copies the PVC data from the restic repository to the protect namespace, which then gets transferred to the user namespace by the controller.
```

Comment:
Should we call this app namespace instead of user namespace? For consistency purposes, as I saw app namespace used previously. wdyt?